scripts/doclingTemp.py

```
@@ -0,0 +1,366 @@
+
+        # ---- Build page markdown ----
+        md_page = build_markdown_page(
+            page_no=page_no_display,
+            content_md=content_md or "",
+            table_md_blocks=tables_markdown_blocks,
+            image_rel_paths=figure_rel_paths,
+        )
+        md_pages.append(md_page)
+
+        # ---- Build JSON page record ----
+        dpi = page._default_image_scale * 72  # exposed by docling’s page
```

Ruff (0.12.2)

Comment contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF003)